cli/release.go

```diff
@@ -149,3 +156,31 @@ func runReleaseAddDocker(args *docopt.Args, client *controller.Client) error {
 
 	return nil
 }
+
+func canonicalizeDockerURI(rawurl string) (string, error) {
+	log.Println("Validating Docker image URI.")
+	u, err := url.Parse(rawurl)
+	if err != nil {
+		return "", err
+	}
+	if u.Host != "registry.hub.docker.com" {
+		return "", fmt.Errorf(`Invalid registry "%s"`, u.Host)
```

Comment:
This check will prevent people from using a private registry. An invalid registry should be determined by the return value of `GetImage()` (it will return an error if the registry does not return valid responses).